src/components/NavBar/SignOutButton.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import IconButton from "@mui/material/IconButton";
+import LogoutIcon from "@mui/icons-material/Logout";
+import { useRouter } from "next/navigation";
+
+const SignOutButton: React.FC<{}> = () => {
+    const router = useRouter();
+
+    const handleSignOutClick = (): void => {
+        fetch("/auth/signout", {
+            method: "POST",
+        }).then(() => {
+            router.push("/login");
+        });
+    };
+
+    return (
+        <IconButton
+            size="medium"
```

Comment:
again size is potentially default (from my very quick Google)

Reply:
It is default size yes